Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -290,6 +290,11 @@ void GlobalObject::put_to_scope(const FlyString& name, Variable variable)
     put(name, variable.value);
 }
 
+bool GlobalObject::delete_from_scope(const FlyString& name)
```

Comment:
Please use east const style in new code :)

Reply:
sorry, fixed!